addons/csw/functions/fnc_assemble_canDeployTripod.sqf

```diff
@@ -0,0 +1,21 @@
+/*
+ * Author: TCVM
+ * Checks if the player can deploy the tripod.
+ *
+ * Arguments:
+ * 0: Unit <OBJECT>
+ *
+ * Return Value:
+ * Can deploy <BOOL>
+ *
+ * Example:
+ * [player] call ace_csw_fnc_canDeployTripod
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
```

Comment:
As much as I know include must be before header to fix line numbers reported in case of errors.